packages/core/src/cli/lib/config-file-updater.ts

```diff
@@ -128,32 +131,59 @@ export async function updateConfigFile(
       }
       if (!nodeToUpdate) throw new Error('Unable to find `export default` or `module.exports = `');
 
-      // if a path was passed in, we'll try to find it in an object
-      if (singleUpdate.path) {
-        // if the node is a function call we'll dive into it and assume we want the first arg
-        // (this matches the `export default defineConfig({...})` pattern that many config files use)
-        if (nodeToUpdate.type === 'CallExpression' && nodeToUpdate.arguments.length) {
-          nodeToUpdate = nodeToUpdate.arguments[0];
-        }
-        if (nodeToUpdate.type !== 'ObjectExpression') {
-          throw new Error('Expected to find an object node to use apply the path selector');
-        }
-        // currently only supports path of depth 1, but should support going deeper
-        pathNodeToUpdate = nodeToUpdate.properties.find((n) => n.type === 'Property' && (n.key as any).name === singleUpdate.path![0]);
-        if (pathNodeToUpdate && pathNodeToUpdate.type !== 'Property') {
-          throw new Error('Node is not a property');
-        }
-      }
+    // this looks for a function call by name
+    } else if (singleUpdate.symbol.endsWith('()')) {
```

Comment:
previously we only handled modifying the default export, now we can also just look for a function call by name. This is needed to mess with the `remix()` vite plugin options